ams/server/src/main/java/com/netease/arctic/server/catalog/InternalCatalog.java

```diff
@@ -135,21 +135,17 @@ public boolean exist(String database, String tableName) {
   }
 
   private String getDatabaseDesc(String database) {
-    return new StringBuilder()
-        .append(name())
-        .append('.')
-        .append(database)
-        .toString();
+    return name() +
```

Comment:
The project use 2 spaces as indent and tab size, you may reconfigure it an reformat the codes here.